public/app/plugins/datasource/cloudwatch/components/LogsQueryField.test.tsx

```diff
@@ -1,35 +0,0 @@
-import { render, screen, waitFor } from '@testing-library/react';
-import _, { DebouncedFunc } from 'lodash'; // eslint-disable-line lodash/import-scope
-import React from 'react';
-
-import { ExploreId } from '../../../../types';
-import { setupMockedDataSource } from '../__mocks__/CloudWatchDataSource';
-
-import CloudWatchLogsQueryField from './LogsQueryField';
-
-jest
-  .spyOn(_, 'debounce')
-  .mockImplementation((func: (...args: any) => any, wait?: number) => func as DebouncedFunc<typeof func>);
-
-describe('CloudWatchLogsQueryField', () => {
-  it('loads defaultLogGroups', async () => {
```

Comment:
This test didn't do much - defaults are now defined in the [datasource](https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/cloudwatch/datasource.ts#L180-L185) file and applied by the framework, so testing this cannot be done within the scope of the plugin. It would require an e2e test of something to verify this. 

Aside from removing this test, this PR doesn't change things in the LogQueryField so don't know what tests can be added here. Think it's better to remove the test file than keeping an empty file. WDYT?